src/main/java/LacksDataBaseProject/SetLack.java

```diff
@@ -0,0 +1,16 @@
+package LacksDataBaseProject;
+
+public class SetLack {
+    public static void setDate(){
+
+    }
+    public static void setComment(){
+
+    }
+    public static void setOrderedAmount(){
+
+    }
+    public static void changePurchaser(){
+
```

Comment:
To jest takie glupie miejsce - bo zgodnie z oczekiwaniami "Klienta" tu ma byc taka mozliwosc, ze Purchaserrecznie sobie zmieni Purchaser'a w Lack'u na innego (w wyjątkowej sytuacji), ale na liscie Supplierów wszystko zostanie tak jak było. Jeszcze nie myslalem jak to dokladnie zrobic, ale ci sygnalizuje.